src/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -0,0 +1,145 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+import type { HTMLProps } from "react";
+
+export type Props = {
+  className?: string;
+  blocks: Array<CodeSnippetBlockProps>;
```

Comment:
We tend to use the CodeSnippetBlockProps[] syntax for typing arrays.

Reply:
Ah, sure, that's nicer, forgot about that syntax.